action.cjs

```diff
@@ -0,0 +1,221 @@
+const fs = require('fs')
+
+function hashFiles (filename) {
+  return fs.existsSync(filename)
+}
+
+const { spawn } = require('child_process')
```

Comment:
prefer to put the `requires` together at the top of the file

Reply:
imports too?

Reply:
generally yes unless they are being lazy-loaded on purpose (like not loaded unless some condition passes)